### PR TITLE
#448 Use python image in drone

### LINKED ISSUE #446: CI.md:25-26: Try to rm --no-cache build flags from drone...
The puzzle `303-7eff8043` from #303 has to be resolved:

https://github.com/fidals/shopelectro/blob/380d159b411e97c655c666f44fe876715b9f1834/CI.md#L25-L26

The puzzle was created by duker33 on 25-Jul-18. 

Estimate: 30 minutes,     

If you have any technical questions, don't ask me, submit new tickets instead. The task will be "done" when the problem is fixed and the text of the puzzle is _removed_ from the source code. Here is more about [PDD](http://www.yegor256.com/2009/03/04/pdd.html) and [about me](http://www.yegor256.com/2017/04/05/pdd-in-action.html).

### LINKED ISSUE #448: .drone.yml:9-12: Use ready image for python in drone....
The puzzle `352-4d0775ee` from #352 has to be resolved:

https://github.com/fidals/shopelectro/blob/497ad6c4c65b6363fe6ee786a8b68be29d9e6783/.drone.yml#L9-L12

The puzzle was created by duker33 on 25-Jul-18. 

Estimate: 60 minutes,     

If you have any technical questions, don't ask me, submit new tickets instead. The task will be "done" when the problem is fixed and the text of the puzzle is _removed_ from the source code. Here is more about [PDD](http://www.yegor256.com/2009/03/04/pdd.html) and [about me](http://www.yegor256.com/2017/04/05/pdd-in-action.html).

### PR DESCRIPTION
**Pull request by duker33**
Closes #446 
Closes #448

### CODE REVIEW

File: .drone.yml [hunk 1]
```diff
@@ -38,16 +24,27 @@ pipeline:
...
+  python-build:
+    image: docker/compose:1.22.0
+    commands:
+      - cd docker/
+      # Build python images with sources and static files
+      - docker-compose -f docker-compose-build.yml build python-dev python-prod
+    volumes:
+      - /var/run/docker.sock:/var/run/docker.sock
+    when:
+      event: push
```
Comment (artemiy312) [inline .drone.yml]: `event: push` -> `event: [push, pull_request]`

File: .drone.yml [hunk 2]
```diff
@@ -38,16 +24,27 @@ pipeline:
...
-      - cp -r front_build/ /drone/shopelectro
     when:
       event: [push, pull_request]
 
+  python-build:
+    image: docker/compose:1.22.0
+    commands:
+      - cd docker/
+      # Build python images with sources and static files
+      - docker-compose -f docker-compose-build.yml build python-dev python-prod
```
Comment (artemiy312) [inline .drone.yml]: >build python-dev python-prod

`python-prod` should be built on `push` event and  `python-dev` both on `push` and `pull_request`, so let's build them separately

File: docker/docker-compose.yml [hunk 3]
```diff
@@ -44,8 +44,6 @@ services:
       #- ../gulpfile.babel.js:/usr/app/src/gulpfile.babel.js
       - ../front:/usr/app/src/front
       - ../front_build:/usr/app/src/front_build
-      # @todo #457:15m Return back refarm deps to nodejs service.
```
Comment (artemiy312) [inline docker/docker-compose.yml]: It is not a relevant issue anymore?

File: .drone.yml (hunk 1 above)
Comment (duker33) [reply .drone.yml]: Done.

**Comment by duker33** [comment]
seems Artemiy agreed with my comment. So, i'll merge it

**Pull request merged**